oscarlab/graphene
# [Meson] Compile LibOS

## Description of the changes 

This moves compiling `libsysdb.so` from `Makefile`s to `meson.build` files.

## How to test this PR? 

Build should succeed, and tests should pass. Specifically LibOS tests should be unaffected.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene/2351)
